3.0.0/README.md

```diff
@@ -88,37 +192,39 @@ An array of data files in GeoJSON format. {z}, {x} and {y}, if present, are repl
 }
 ```
 
-## 3.5 `description`
+## 3.8 `description`
 
 OPTIONAL. Default: null.
 
-A text description of the tileset. The description can contain any valid unicode character as described by the JSON specification [RFC 8259](https://tools.ietf.org/html/rfc8259).
+A text description of the set of tiles. The description can contain any valid unicode character as described by the JSON specification [RFC 8259](https://tools.ietf.org/html/rfc8259).
 
 ```JSON
 {
   "description": "We are volcanoes. When we women offer our experience as our truth, as human truth, all the maps change. There are new mountains."
 }
 ```
 
-## 3.6 `fillzoom`
+## 3.9 `fillzoom`
 
 OPTIONAL. Default: null.
 
-An integer specifying the zoom level at which to generate overzoomed tiles from. Implementations may generate overzoomed tiles from parent tiles if the requested zoom level does not exist. In most cases, overzoomed tiles are generated from the maximum zoom level of the tileset. If fillzoom is specified, the overzoomed tile is generated from the fillzoom level.
+An integer specifying the zoom level at which to generate overzoomed tiles from. Implementations may generate overzoomed tiles from parent tiles if the requested zoom level does not exist. In most cases, overzoomed tiles are generated from the maximum zoom level of the set of tiles. If fillzoom is specified, the overzoomed tile is generated from the fillzoom level.
```

Comment:
This attribute is really confusing and surprising to me. What's the use case? Why would you want to overzoom from a level other than the maximum level? Does a client have to respect this value? What if it doesn't? 

In any case, I'd also suggest that this wording is unclear about who or what is doing the overzooming: 

> ... at which to generate overzoomed tiles from. Implementations may generate overzoomed tiles...

"Implementation" is too vague, because it could refer to either the server or the client. And particularly when describing "generating tiles", which sounds more like something the server does (although I understand here we mean the client).

Suggest:

`which a client should use for generating overzoomed tiles, when tiles are not available at a requested zoom level.`

>In most cases, overzoomed tiles are generated from the maximum zoom level of the set of tiles. If fillzoom is specified, the overzoomed tile is generated from the fillzoom level.

I find it confusing that we are describing the behaviour of clients here. Is this a descriptive ("clients tend to do the following") or prescriptive ("clients should do the following") statement? I don't really understand its intent. Maybe something like this would be better:

> If no `fillzoom` is set, clients MAY use the `maxzoom` setting to determine which tiles to use for overzooming.

But maybe these two sentences just aren't needed. (Or could be replaced by changing "Default: null" to "Default: maxzoom".)

Reply:
@stevage this has been one of the more difficult fields to describe - thanks for your feedback! `fillzoom` is pretty complex. Here's my thinking out loud - curious of our mental models match: 

My understanding (which doesn't include a lot of historical information) is that fillzoom is particularly helpful for tile requests _within_ the min/maxzoom range where the tile does not exist. I've been thinking of "implementation" from a server-side perspective, rather than a client. So for example, take the following tilejson and a server retrieving a tile at z10: 

```
{
  "minzoom": 5,
  "maxzoom": 11,
  "fillzoom": 8
}
```

Assuming the z10 tile you request isn't available (just no data there, so no tile), the fillzoom gives the server an opportunity to quickly overzoom from the z8 parent tile, rather than working up the tile pyramid one parent at a time (i.e. request z9 first, then z8 if z9 does not exist as well). So therefore fillzoom requires a bit of deeper knowledge about the data in the vector tiles, such that the curator (person making the tiles) knows that their fillzoom choice is representative of data availability in the set of tiles.

> > In most cases, overzoomed tiles are generated from the maximum zoom level of the set of tiles. If fillzoom is specified, the overzoomed tile is generated from the fillzoom level.

> I find it confusing that we are describing the behaviour of clients here. Is this a descriptive ("clients tend to do the following") or prescriptive ("clients should do the following") statement?

Agree this is confusing from a client perspective. Would it be more clear if we specified this is a _descriptive_ statement from a server perspective? This makes me consider that servers can technically do whatever they want with this information, meaning fillzoom is not only OPTIONAL by the specification, but likely OPTIONAL for a server to use even if it exists.

I'll refrain from making changes to this field for now, so we can keep working on a better description here!